test/kafo/hook_context_test.rb

```diff
@@ -43,5 +44,16 @@ module Kafo
         assert_equal context.module_enabled?('known_module'), true
       end
     end
+
+    describe "#app_option?" do
+      specify do
+        app = {'known_option' => 'option_arg'}
+        config = Minitest::Mock.new
+        kafo.expect :config, config
+        config.expect :app, app
+        assert_equal context.app_option?('known_option'), true
+        assert_equal context.app_option?('unknown_option'), false
```

Comment:
Nit: In general it's considered a bad practice to combine multiple tests into a single `specify`. One is testing for the positive case, the other for the negative.